horse/users.py

```diff
@@ -9,6 +9,21 @@
 users_api = Api(users_bp)
 
 
+def get_user_by_id(user_id):
+    return [u for u in users if u['id'] == user_id][0]
```

Comment:
yep, let's leave it until we have at least repositories and or schema validation